modules/apps/bookmarks/bookmarks-service/src/main/java/com/liferay/bookmarks/service/base/change/tracking/spi/reference/BookmarksEntryTableReferenceDefinition.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.bookmarks.service.base.change.tracking.spi.reference;
```

Comment:
The package should be `com.liferay.bookmarks.internal.change.tracking.spi.reference`